src/AI/agents/agents/graph/nodes/intake_node.py

```
@@ -0,0 +1,140 @@
+
+        sink.send(
+            AgentEvent(
+                type="plan_proposed",
+                session_id=state.session_id,
+                data={"plan": state.step_plan[0], "step": state.step_plan[0]},
+            )
+        )
+
+        state.next_action = "scan"
+
+    except Exception as exc:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)